tools/partition.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import argparse
+import signal
+import dgl
+from dgl import backend as F
+from dgl.data.utils import load_graphs, save_graphs
+
+def main():
```

Comment:
I'm a little confused about the DGL_KE data preprocessing pipeline. The metis partition API needs a DGLGraph as input. However, the DGLGraph will be constructed by the dataloader. So we will invoke the dataloader before preprocessing?

Reply:
yes, we call data loader to construct DGLGraph and then invoke metis partitioning to partition the graph.